### PR TITLE
Add Coodesker

### PR DESCRIPTION
**Pull request by chriscp-cat**

### CODE REVIEW

File: bucket/Coodesker-portable.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "version": "1.0.1.4",
+    "description": "自动分类整理桌面上的快捷方式、目录、图片等文件",
```
Comment (Ryanjiena) [inline bucket/Coodesker-portable.json:3]:
```suggestion
    "description": "Coodesker helps you organize your PC by automatically placing your shortcuts and icons into resizable shaded areas on your desktop called boxes. Coodesker will always be free to users.",
```

File: bucket/Coodesker-portable.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "homepage": "https://www.coodesker.com",
+    "license": {
+        "identifier": "Freeware"
+    },
+    "notes": "We don't persist your Coodesker data, they are still storaged in '$env:APPDATA\\coodesker'.",
```
Comment (Ryanjiena) [inline bucket/Coodesker-portable.json:8]:
```suggestion
    "notes": "We don't persist your Coodesker data, they are still storaged in '$env:APPDATA\\Coodesker'.",
```

File: bucket/Coodesker-portable.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/coodesker/coodesker-desktop/releases/download/v1.0.1.4/Coodesker-Portable-x64_1.0.1.4.zip",
+            "hash": "e7d3375bd834bfec5165628857c415059e1012a2023b4bffd8607495637d9c0b",
```
Comment (Ryanjiena) [inline bucket/Coodesker-portable.json:12]:
```suggestion
            "hash": "e7d3375bd834bfec5165628857c415059e1012a2023b4bffd8607495637d9c0b",
            "bin": "coodesker-x64.exe",
```

File: bucket/Coodesker-portable.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            "shortcuts": [
+                [
+                    "coodesker-x64.exe",
+                    "Coodesker"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/coodesker/coodesker-desktop/releases/download/v1.0.1.4/Coodesker-Portable-x86_1.0.1.4.zip",
+            "hash": "096a5d8ab33b28eb1273542f375b54902b33a498569b573ab11a934c1076901f",
```
Comment (Ryanjiena) [inline bucket/Coodesker-portable.json:22]:
```suggestion
            "hash": "096a5d8ab33b28eb1273542f375b54902b33a498569b573ab11a934c1076901f",
            "bin": "coodesker.exe",
```

File: bucket/Coodesker-portable.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                    "Coodesker"
+                ]
+            ]
+        }
+    },
+    "checkver": {
+        "url": "https://www.coodesker.com/coodesker-desktop/release.json",
+        "jsonpath": "$.version",
+        "regex": "^v([\\.\\d]*)",
+        "replace": "$1"
```
Comment (Ryanjiena) [inline bucket/Coodesker-portable.json:35]:
```suggestion
        "github": "https://github.com/coodesker/coodesker-desktop"
```

**Review by Ryanjiena — CHANGES_REQUESTED** [review]

**Comment by JaimeZeng** [comment]
All changes look good.

Wait for review from human collaborators.
### Coodesker-portable

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Pull request merged**